sqlalchemy_continuum/manager.py

```diff
@@ -431,6 +431,21 @@ def track_cloned_connections(self, c, opt):
                 if not connection.closed and connection.connection is c.connection:  # ConnectionFairy is the same - this is a clone
                     self.units_of_work[c] = uow
 
+    def get_python_processed_parameter(self, parameter, column, dialect):
+        param_transformer = column.type.dialect_impl(dialect)
+        process = param_transformer.result_processor(dialect, column.type)
+        if process is not None:
+            # https://github.com/sqlalchemy/sqlalchemy/blob/231262d4782fc4d1a5d5fdbf985a06e04270e878/lib/sqlalchemy/sql/type_api.py#L452
```

Comment:
When adding a link unless you explicitly need this commit - you could just use the last stable version:
https://github.com/sqlalchemy/sqlalchemy/blob/rel_1_4_40/lib/sqlalchemy/sql/type_api.py#L335
THe link becomes easier to read also.
Plus you're linking to the function there - it would be ideal to explain what you want us to read there.
For example:
```
If `result_processor` is None - no processing is required as per sqlalchemy
```

Reply:
Sure will update this one and keep that in check for future PR(s)